explorer/explorerroutes.go

```diff
@@ -140,30 +143,90 @@ func (exp *explorerUI) DisapprovedBlocks(w http.ResponseWriter, r *http.Request)
 	io.WriteString(w, str)
 }
 
+// show only regular tx in block.Transactions, exclude coinbase (reward) transactions
+// for use in NextHome handler and websocket response to getmempooltxs event
+func filterRegularTx(txs []*TrimmedTxInfo) []*TrimmedTxInfo {
+	transactions := make([]*TrimmedTxInfo, 0)
+	for _, tx := range txs {
+		if !tx.Coinbase {
+			transactions = append(transactions, tx)
+		}
+	}
+	return transactions
```

Comment:
A pattern I prefer for functions like this is to use a named return parameter and do no `make`.  i.e.:

```go
func filterRegularTx(txs []*TrimmedTxInfo) (transactions []*TrimmedTxInfo) {
	for _, tx := range txs {
		if !tx.Coinbase {
			transactions = append(transactions, tx)
		}
	}
	return
}
```

When appropriate, named parameters are nice as they are self-documenting, even if it's already clear in this case.  But when the slice is OK to be nil, it also eliminates the unnecessary make or other declaration of the variable.